Childrens-Social-Care-CPD/Contentful/CustomHtmlHelpers.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Contentful.Core.Models;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace Childrens_Social_Care_CPD.Contentful;
+
+public static class CustomHtmlHelpers
+{
+    public static IHtmlContent ContentLink(this IHtmlHelper helper, string text, string uri)
+    {
+        if (helper == null) return null;
+
+        var tagBuilder = new TagBuilder("a");
+        
+        var href = uri switch
```

Comment:
Ive been out too long, this syntax rocks :)

Reply:
Switch pattern matching. Something I'd wanted in C# for years.